src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationClientFactory.cs

```diff
@@ -3,20 +3,32 @@
 //
 using Azure.Core;
 using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.ConfigurationClients;
 using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
 {
     internal class ConfigurationClientFactory : IConfigurationClientFactory
```

Comment:
We should either leave the `ConfigurationClientFactory` as is or rename it to `ConfigurationClientsFactory` or  `ConfigurationClientCollectionFactory` if we really need it.

Reply:
IMO, I think you can leave this class as is and let AACSource handle the extra logic of the ConfiguraitonClient collection.